<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-QMQ0VG44" ignored="0" name="Test(failed)" tests="2" failures="2" timestamp="24 Sep 2020 07:28:36 GMT" time="14.83" errors="0">
  <testcase name="editBoxDemo" time="14.759" classname="extentreport_demo.Test3">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at extentreport_demo.Test3.editBoxDemo(Test3.java:24)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- editBoxDemo -->
  <testcase name="editBoxDemo" time="14.799" classname="extentreport_demo.Test3">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at extentreport_demo.Base.getScreenshotPath(Base.java:27)
at extentreport_demo.Listeners.onTestFailure(Listeners.java:51)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- editBoxDemo -->
</testsuite> <!-- Test(failed) -->
